autoload/mysql.vim

```diff
@@ -136,3 +136,76 @@ function! mysql#DeleteRow()
         call sqhell#ExecuteCommand(query)
     endif
 endfunction
+
+function! mysql#EditRow()
+    let row = getline('.')
+    let csv = sqhell#CreateCSVFromRow(row)
+    let savecur = getcurpos()
+    let head = sqhell#GetTableHeader()
+    call setpos('.', savecur)
+    let list = sqhell#GetTableName()
+    let table = list[0]
+    let db = list[1]
+
+    :bd
+    new
+    let b:type = 'edit'
+    let b:prev = csv
+    let t:tabInfo = db . '.' . table
+    call append(0, head)
+    call append(1, csv)
+
+    setlocal buftype=nofile
```

Comment:
Can you extract this out into a function and make sure that `sqhell#InsertResultsToNewBuffer` also calls it?

It should take a filetype as an argument to set the buffer to :)

Reply:
In fact can you not just remove this call and use `InsertResultsToNewBuffer`?
That's what it's for